doc/latex/csvsimple/csvsimple-l3.tex

```diff
@@ -723,7 +723,7 @@ \section{Macros for the Processing of CSV Files}\label{sec:makros}%
   Typesets the current data line number. This is the
   current number of accepted data lines without the header line.
   Despite of the name, there is no associated \LaTeX\ counter |csvrow|,
-  but \refCom{thecsvrow} is an accessor the \LaTeX3 integer
+  but \refCom{thecsvrow} being an accessor of the \LaTeX3 integer
   \refCom{g_csvsim_row_int}.
```

Comment:
For the three similar occurrences:
- the first two seem to lack an "of": is an accessor the LaTeX3 integer -> is an accessor **of** the LaTeX3 integer
- the third one was changed in e32ef0539140d6cfa79caf633fe50f9db1d9842d, hence different from the other two. Maybe you want to adapt the first two similarly.
    ```diff
    @@ -754,7 +755,7 @@ \section{Macros for the Processing of CSV Files}\label{sec:makros}%
       current number of all data lines including the header line and all
       lines filtered out.
       Despite of the name, there is no associated \LaTeX\ counter |csvinputline|,
    -  but \refCom{thecsvinputline} is an accessor the \LaTeX3 integer
    +  but \refCom{thecsvinputline} accesses the \LaTeX3 integer
       \refCom{g_csvsim_inputline_int}.
     \begin{dispExample}
     \csvreader[
    ```

Reply:
Yes, you are right. I will adapt the first two accordingly.

Reply:
Just for the record, the first two were adapted in [v2.5.0](https://github.com/T-F-S/csvsimple/releases/tag/v2.5.0) (3cb41f7927ea815d0341226ac9d43bbecaaaf9f4). Thanks.